helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/SkipDecoder.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+import java.io.IOException;
+import org.apache.avro.Schema;
+import org.apache.avro.io.Decoder;
+
+/**
+ * an abstract class that extends from Decoder that helps CachedResolvingDecoder
+ * perform various methods like original ResolvingDecoder.
+ */
+public abstract class SkipDecoder extends Decoder {
+
+  public int readStringSize() throws IOException {
```

Comment:
im assumming this reads the size of a string and is intended to help "skip" the actual string? if so  - it needs docs and also why have a default implementation at all?

Reply:
will change all methods as abstract